lib/ansiblelint/cli.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8; -*-
+import optparse
+import os
+import sys
+
+import ansiblelint
+from ansiblelint.version import __version__
+import yaml
+
+
+__PATH_VARS = ['exclude_paths', 'rulesdir', ]
+
+
+def abspath(path, base_dir):
+    """Make relative path absolute relative to given directory.
+
+    Args:
+       path (str): the path to make absolute
+       base_dir (str): the directory from which make relative paths
+           absolute
+       default_drive: Windows drive to use to make the path
+           absolute if none is given.
+    """
+    if not os.path.isabs(path):
+        # Don't use abspath as it assumes path is relative to cwd.
+        # We want it relative to base_dir.
+        path = os.path.join(base_dir, path)
+
+    return os.path.normpath(path)
+
+
+def expand_to_normalized_paths(config, base_dir=None):
+    base_dir = base_dir or os.getcwd()
+    for paths_var in __PATH_VARS:
+        if paths_var not in config:
+            continue  # Cause we don't want to add a variable not present
+
+        normalized_paths = []
+        for path in config.pop(paths_var):
+            normalized_path = abspath(path, base_dir=base_dir)
+
+            normalized_paths.append(normalized_path)
+
+        config[paths_var] = normalized_paths
+
+
+def load_config(config_file):
+    config_path = config_file if config_file else ".ansible-lint"
+    config_path = os.path.abspath(config_path)
+
+    if os.path.exists(config_path):
+        config_dir = os.path.dirname(config_path)
+        with open(config_path, "r") as stream:
+            try:
+                config = yaml.safe_load(stream)
+
+            except yaml.YAMLError:
+                pass
```

Comment:
If you return here, else-block indent won't be needed
```suggestion
                return
```